### PR TITLE
Added package.json

### LINKED ISSUE #93: Failed to feetch plugin
I have the following error when adding the plugin

Error: Failed to fetch plugin https://github.com/macdonst/SpeechRecognitionPlugin via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: cmd: Command failed with exit code 4294963228 Error output:
.
.
.

### PR DESCRIPTION
**Pull request by jcesarmobile**
Used phonegap-plugin-speech-recognition as the name on the package.json, also changed the id to be the same (all the cordova ones were already picked, but feel free to change it if you don't like it)

Bumped minor version

Changed plugin.xml license to MIT as it's the license file used.

Closes #93 & #91

### CODE REVIEW

**Comment by mosca90** [comment]
@jcesarmobile when you will merge this fix? Thx.

**Comment by jcesarmobile** [comment]
@mosca90 I'm waiting for a review from @macdonst

**Comment by jcesarmobile** [comment]
I thought commenting "Closes #93 & #91" would close the issues once the PR is merged, but doesn't work

**Comment by macdonst** [comment]
Nope you have to do it manually. I'm going to publish to npm sometime today if the wifi is good enough.

**Review by sins921 — APPROVED** [review]

**Pull request merged**
